### ISSUE TITLE
Abnormal circulating ketone body concentrations

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6017]
**Preferred term label:** Abnormal circulating ketone body concentrations

**Synonyms**

**Definition (free text, please give PubMed ID)** Any deviation form the normal concentration of a ketones in the blood circulation., PMID: 24706027

**Parent term (use hpo.jax.org/app)** Abnormal circulating metabolite concentration HP:0032180

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Elevated circulating acetoacetic acid concentration

**Synonyms** Elevated circulating diacetic acid in blood

**Definition (free text, please give PubMed ID)** Concentration of acetoacetic acid in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating ketone concentrations

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Elevated circulating ketone body concentration is a synoynm with https://hpo.jax.org/app/browse/term/HP:0410175

**Comment by pnrobinson** [comment]
3-methyl-2-oxovaleric is not a ketone body to my knowledge

**Comment by pnrobinson** [comment]
added Elevated circulating acetoacetic acid concentration
